app.rb

```diff
@@ -0,0 +1,132 @@
+require './book'
+require './persons'
+require './rental'
+require './student'
+require './teacher'
+require './classroom'
+
+class App
+  def initialize
+    @book = []
+    @persons = []
+    @rentals = []
+  end
+
+  def open_console
+    puts 'Hello There, Welcome to our School library'
+    until list_of_options
+      input = gets.chomp
+      if input == '7'
+        puts 'Thank You for using my School Library!'
+        break
+      end
```

Comment:
You seem to have referred to App's `open_console` as `start_console`. Kindly reconcile the method names so as to avoid the error below:

![image](https://user-images.githubusercontent.com/20554583/216769266-ef7db7e4-410b-4411-be42-57c7adc12d9b.png)